.github/workflows/tests.yml

```diff
@@ -0,0 +1,23 @@
+name: tests
+
+on: pull_request
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install gnome-keyring
+        run: sudo apt-get install gnome-keyring
```

Comment:
I request we remove this step if its not needed, otherwise add a comment about why its needed.

I'm allergic to unnecessary changes getting in, then persisting through legacy, because as those build up it makes it more difficult for newcomers to filter through that kind of distraction.

Reply:
Comment added.